front/app/modules/commercial/analytics/admin/components/RegistrationsCard/Chart.tsx

```diff
@@ -18,27 +17,41 @@ import { toThreeLetterMonth } from 'utils/dateUtils';
 import { generateEmptyData } from './generateEmptyData';
 
 // typings
-import { Moment } from 'moment';
-import { IResolution } from 'components/admin/ResolutionControl';
+import { ProjectId, Dates, Resolution, Layout } from '../../typings';
 import { LegendItem } from 'components/admin/Graphs/_components/Legend/typings';
 import { TimeSeries } from '../../hooks/useRegistrations/typings';
+import { Margin } from 'components/admin/Graphs/typings';
 
-interface Props {
-  timeSeries: TimeSeries | NilOrError;
-  projectFilter?: string;
-  startAtMoment: Moment | null | undefined;
-  endAtMoment: Moment | null | undefined;
-  resolution: IResolution;
-  innerRef: React.RefObject<any>;
-}
+type Props = ProjectId &
+  Dates &
+  Resolution & {
+    timeSeries: TimeSeries | NilOrError;
+    innerRef: React.RefObject<any>;
+    layout?: Layout;
+  };
+
```

Comment:
Standardized our props a bit like this to make sure we always have exactly the same types. E.g. sometimes we had `endAtMoment: Moment | null | undefined`, sometimes only `Moment | null`